e2e/inventory/inventory.go

```diff
@@ -114,10 +114,11 @@ func NewMultiAppBackupAndRestoreTest() Test {
 
 func MultiAppTest() Test {
 	return Test{
-		Name:        "multi-app-install",
-		TestimSuite: "multi-app-install",
+		ID:          "multi-app-install",
+		Name:        "Multi App Install",
 		Namespace:   "multi-app-install",
-		UpstreamURI: "multi-app-install/automated",
+		AppSlug:     "mutli-app-install",
+		UpstreamURI: "mutli-app-install/automated",
```

Comment:
```suggestion
		AppSlug:     "multi-app-install",
		UpstreamURI: "multi-app-install/automated",
```

Reply:
@sgalsaleh this looks like a typo - but i see the license and playwright tests are looking for `mutli` as well so not positive.

Reply:
yeah, it's a typo in the app slug, but this matches so it should work...